test/animationinfos_test.cpp

```diff
@@ -53,29 +53,29 @@ struct RenderingData : TestData {
 void RunAnimationTest(int numFrames, int delay, AnimationDistributionParams params, int numSkippedFrames, int distributeFramesBeforeFrame, const std::vector<TestData *> &vecTestData)
 {
 	const int pnum = 0;
-	PlayerStruct *pPlayer = &plr[pnum];
-	NewPlrAnim(pnum, nullptr, numFrames, delay, 128, params, numSkippedFrames, distributeFramesBeforeFrame);
+	AnimationInfos animInfo;
```

Comment:
Should this be:
```cpp
	AnimationInfos animInfo = plr[pnum].AnimInfos;
```
?

If so, wouldn't that also be bad, since we'd be creating a whole copy of the entire struct in the stack? Shouldn't we be using `const AnimationInfos&` (for immutable) or `AnimationInfos*` (for mutable)?

Reply:
This UnitTest only thest `AnimationInfo`.
And `AnimationInfo` will be used in other places too.
Thats why I dropped using the `PlayerStruct`.